api/src/services/translations.ts

```diff
@@ -0,0 +1,47 @@
+import type { Item, PrimaryKey } from '@directus/types';
+import getDatabase from '../database/index.js';
+import { ItemsService } from './items.js';
+import { InvalidPayloadException } from '../index.js';
+import type { AbstractServiceOptions } from '../types/services.js';
+import type { MutationOptions } from '../types/items.js';
+
+export class TranslationsService extends ItemsService {
+	constructor(options: AbstractServiceOptions) {
+		super('directus_translations', options);
+
+		this.knex = options.knex || getDatabase();
+		this.accountability = options.accountability || null;
+		this.schema = options.schema;
+	}
+
+	private async translationKeyExists(key: string, language: string) {
+		const result = await this.knex.select('').from(this.collection).where({ key, language });
```

Comment:
Lets play it safe here:

```suggestion
		const result = await this.knex.select('id').from(this.collection).where({ key, language });
```

Gut feeling says there's _probably_ an edge case bug waiting to happen where one of the DBs doesn't support fetching nothing